state-chain/pallets/cf-validator/src/lib.rs

```diff
@@ -399,8 +398,6 @@ pub mod pallet {
 					let num_primary_candidates = rotation_state.num_primary_candidates();
 					match T::KeyRotator::status() {
 						AsyncResult::Ready(KeyRotationStatusOuter::KeyHandoverComplete) => {
-							let new_authorities = rotation_state.authority_candidates();
-							HistoricalAuthorities::<T>::insert(rotation_state.new_epoch_index, new_authorities);
```

Comment:
I removed these 2 lines because I think this is redundant, we add the old set to the HistoricalAuthorities once we complete the rotation inside `initialise_new_epoch` function.